wdio.conf.js

```diff
@@ -64,13 +66,15 @@ const config = {
     google: ['./src/google/**/*.spec.ts'],
   },
 
-  beforeTest: function(test) {
-    browser.windowHandleSize({ width: 1280, height: 1024 });
+  beforeSession: (_config, _capabilities, specs) => {
+    if (!specs.length) {
+      return;
+    }
     if (!flags['replay-fixtures'] && !flags['record-fixtures']) {
       return;
     }
     const fixtureService = new FixtureService();
-    testRun = { fixtureFile: fixtureService.fixturePathForTestPath(test.file) };
+    testRun = { fixtureFile: fixtureService.fixturePathForTestPath(specs[0]) };
```

Comment:
We used to have a hook into test file names via the `beforeTest` method; however, the API has changed, and `test` no longer include file information. We are able to get it in the `beforeSession` call, as there's always just one entry in `specs`, and it happens to be the absolute file name of the test.

It's possible this doesn't work in some scenario, but it works in all the ones we have, which is good enough for me?